src/main/java/supersymmetry/common/metatileentities/multi/electric/MetaTileEntityClarifier.java

```diff
@@ -0,0 +1,86 @@
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.World;
+import supersymmetry.api.recipes.SuSyRecipeMaps;
+import supersymmetry.common.blocks.BlockDrillHead;
+import supersymmetry.common.blocks.BlockMultiblockTank;
+import supersymmetry.common.blocks.SuSyBlocks;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+
+import static gregtech.api.util.RelativeDirection.*;
+
+public class MetaTileEntityClarifier extends RecipeMapMultiblockController {
+    public MetaTileEntityClarifier(ResourceLocation metaTileEntityId) {
+        super(metaTileEntityId, SuSyRecipeMaps.CLARIFIER);
+        this.recipeMapWorkable = new MultiblockRecipeLogic(this, true);
+    }
+
+    public MetaTileEntity createMetaTileEntity(IGregTechTileEntity tileEntity) {
+        return new MetaTileEntityClarifier(this.metaTileEntityId);
+    }
+
+    protected BlockPattern createStructurePattern() {
+        return FactoryBlockPattern.start()
+                .aisle("      AAAA      ", "      AAAA      ", "      AAAA      ", "                ")
+                .aisle("    AAAAAAAA    ", "    AADDDDAA    ", "    AA    AA    ", "                ")
+                .aisle("   AAAAAAAAAA   ", "   ADDDDDDDDA   ", "   A        A   ", "                ")
+                .aisle("  AAAAAAAAAAAA  ", "  ADDDDDDDDDDA  ", "  A          A  ", "                ")
+                .aisle(" AAAAAAAAAAAAAA ", " ADDDDDDDDDDDDA ", " A            A ", "                ")
+                .aisle(" AAAAAAAAAAAAAA ", " ADDDDDDDDDDDDA ", " A            A ", "                ")
+                .aisle("AAAAAAAAAAAAAAAA", "ADDDDDDDDDDDDDDA", "A              A", "                ")
+                .aisle("AAAAAAAAAAAAAAAA", "ADDDDDDBBDDDDDDA", "A      BB      A", "       EE       ")
+                .aisle("AAAAAAAAAAAAAAAA", "ADDDDDDBBDDDDDDA", "A      BBF     A", "       EE       ")
+                .aisle("AAAAAAAAAAAAAAAA", "ADDDDDDDDDDDDDDA", "A       FFF    A", "                ")
+                .aisle(" AAAAAAAAAAAAAA ", " ADDDDDDDDDDDDA ", " A       FFF  A ", "                ")
+                .aisle(" AAAAAAAAAAAAAA ", " ADDDDDDDDDDDDA ", " A        FFF A ", "                ")
+                .aisle("  AAAAAAAAAAAA  ", "  ADDDDDDDDDDA  ", "  A        FFA  ", "                ")
+                .aisle("   AAAAAAAAAC   ", "   ADDDDDDDDA   ", "   A        A   ", "                ")
+                .aisle("    AAAAAAAAC   ", "    AADDDDAA    ", "    AA    AA    ", "                ")
+                .aisle("      AAAA BBB  ", "      AAAA BSB  ", "      AAAA      ", "                ")
+                .where('S', selfPredicate())
+                .where('A', states(MetaBlocks.STONE_BLOCKS.get(StoneVariantBlock.StoneVariant.SMOOTH).getState(StoneVariantBlock.StoneType.CONCRETE_LIGHT))
+                        .or(autoAbilities(true, true, true, true, true, true, false)))
```

Comment:
You can just do autoAbilities() here - the muffler mechanics are controlled by another method which needs overriding to enable.